client/Windows/CMakeLists.txt

```diff
@@ -18,6 +18,27 @@
 set(MODULE_NAME "wfreerdp-client")
 set(MODULE_PREFIX "FREERDP_CLIENT_WINDOWS_CONTROL")
 
+if (${CMAKE_GENERATOR} MATCHES "MinGW Makefiles")
```

Comment:
this does not seem right, why is this necessary?

Reply:
Just to change the resource file. Cmake does not support 'if' inside set(), so forced to copy whole list with external if...